controller.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 const (
-	heritageLabel = "heritage"
-	pdbController = "pdb-controller"
+	heritageLabel             = "heritage"
+	pdbController             = "pdb-controller"
+	nonReadyTTLAnnotationName = "pdb-controller.moscar.net/non-ready-ttl"
```

Comment:
I would be fine with using `zalando.org` here instead of `moscar.net` to not confuse users (though this feature is mostly for us). Technically Zalando owns most of this anyway because it has been partly coded at work/is related to work :)